packages/jest-each/src/__tests__/index.test.ts

```diff
@@ -30,4 +30,77 @@ describe('template', () => {
       expect(a + b).toBe(expected);
     });
   });
+
+  describe('comments', () => {
+    describe('removes commented out tests', () => {
+      let testCount = 0;
+      const expectedTestCount = 2;
+
+      each`
+      a    | b    | expected
+      // ${1} | ${1} | ${0}
+      ${1} | ${1} | ${2}
+      /* ${1} | ${1} | ${5} */
+      ${1} | ${1} | ${2}
+    `.test('returns $expected when given $a and $b', ({a, b, expected}) => {
+        testCount += 1;
+        expect(a + b).toBe(expected);
+      });
+
+      test('test runs', () => {
+        expect(testCount).toEqual(expectedTestCount);
+      });
+    });
+
+    describe('removes trailing comments', () => {
+      let testCount = 0;
+      const expectedTestCount = 2;
+
+      each`
+      a    | b    | expected
+      ${0} | ${0} | ${0} // ignores trailing comment
+      ${1} | ${1} | ${2} /* ignores second comment */
```

Comment:
Maybe also a test for `/* /* */` just to make sure the regex "lexer" works correctly

Reply:
I think I've added this in: https://github.com/facebook/jest/pull/8717/commits/fffad4de4d915feab6b9ae57f448c5aaadd8c615.